tutorials/rpc/py/robot_client.py

```diff
@@ -20,11 +20,19 @@ class RobotControlClient(RobotControl, rpc.ClientBase):
     pass
 
 
+async def wait_for_service(client: RobotControlClient):
+    while client.matched_service_count == 0:
+        await sleep(0.1)
+
+
 async def main():
     participant = dds.DomainParticipant(domain_id=0)
     client = RobotControlClient(participant, "MyRobotControl")
 
-    sleep(2)
+    # For versions 7.4.0 and below:
+    await wait_for_service(client)
```

Comment:
I'd prefer to keep the `sleep`. This version of wait_for_service has the race condition we want to avoid. I also want to maintain the tutorial code as simple and short as possible.

Showing the sleep next to the client.wait_for_service also makes it very clear the improvement that 7.5+ provides.